pkg/cmd/pr/view/fixtures/prViewPreviewWithAllChecksFailing.json

```diff
@@ -46,6 +46,7 @@
                   "contexts": {
                     "nodes": [
                       {
+                        "__typename": "CheckRun",
```

Comment:
These are needed because of https://github.com/cli/cli/pull/7462/files#diff-21986411c01246bfb45a509ecd73874c24989888023920096d6bf7481e533b04R324

I was unfortunate that I had to track down why the test was failing when the implementation was correct because the fixture was perhaps overspecified. It represents what the test needs rather than what a response actually looks like. It's a fine balance between _reality_ and _extraneous details_. Something for me to mull over!